IPs/RSKIP191.md

```diff
@@ -0,0 +1,22 @@
+# Remove non Ethereum opcodes from virtual machine
+
+
+|RSKIP          | 191 |
+| :------------ |:-------------|
+|**Title**      |Remove opcodes|
```

Comment:
Title should be full and I think "incompatible" express better than non-Ethereum, because RSK is not Ethereum in several others attributes. 
Proposed title:
Remove from virtual machine opcodes incompatible with Ethereum